rotkehlchen/exchanges/okx.py

```diff
@@ -273,7 +273,7 @@ def query_balances(self, **kwargs: Any) -> ExchangeQueryBalances:
                 continue
 
             try:
-                amount = deserialize_asset_amount(currency_data['availBal'])
+                amount = deserialize_asset_amount(currency_data['availBal'])+deserialize_asset_amount(currency_data['frozenBal'])
```

Comment:
```suggestion
                amount = deserialize_asset_amount(currency_data['availBal']) + deserialize_asset_amount(currency_data['frozenBal'])
```

could you also update `test_okx_query_balances` modifying a `frozenBal` and checking that this change works?

Also what are frozen balances in okx and when do they get frozen?